jobs/config.json

```diff
@@ -7437,7 +7437,7 @@
       "--image-project=ubuntu-os-gke-cloud",
       "--provider=gke",
       "--test_args=--ginkgo.focus=\\[Feature:ClusterSizeAutoscalingScaleUp\\]|\\[Feature:ClusterSizeAutoscalingScaleDown\\] --ginkgo.skip=\\[Flaky\\] --minStartupPods=8",
-      "--timeout=300m"
+      "--timeout=330m"
```

Comment:
420m

Reply:
this should actually be generated by 
```
In $GOPATH/src/k8s.io/test-infra,

  $ bazel run //experiment:generate_tests -- \
      --yaml-config-path=experiment/test_config.yaml \
      --json-config-path=jobs/config.json \
      --prow-config-path=prow/config.yaml

  After generating the tests, you should run:
  $ bazel run //jobs:config_sort
```

Reply:
We have different timeouts for GCE and GKE jobs. The reason for it is because 1) we run more tests in GKE, 2) we tend to trigger a lot of master restarts on GKE (doesn't happen at all in GCE.)